src/Fist/Http/tests/RequestTest.php

```diff
@@ -0,0 +1,139 @@
+<?php
+
+namespace FistTest;
+
+use Fist\Http\Request;
+use Fist\Testing\TestCase;
+
+class RequestTest extends TestCase
+{
+    public function testBuildingInstance()
+    {
+        $request = new Request();
+
+        $this->assertInstanceOf(Request::class, $request);
+    }
+
+    public function testBuildingFromGlobals()
+    {
+        $request = Request::createFromGlobals();
+
+        $this->assertInstanceOf(Request::class, $request);
+    }
+
+    public function testGettingRequestMethod()
+    {
+        $request = Request::createFromGlobals();
+
+        $this->assertEquals('GET', $request->getMethod());
+    }
+
+    public function testGettingFakedRequestMethod()
```

Comment:
__[phpmd]__
* testGettingFakedRequestMethod accesses the super-global variable $_SERVER.

<!-- Comment by [SideCI](https://sideci.com) -->